server/src/setup/http/express/basic-app.ts

```diff
@@ -0,0 +1,23 @@
+import { RequestContext } from '@mikro-orm/core'
+import express, { Router } from 'express'
+import { MikroORM } from '../../db'
+import { App } from './app'
+
+export interface BasicAppOptions {
+  mikroORM?: MikroORM
```

Comment:
Does the following code suggestion make sense? In the future we could change the type to `orm: MikroORM | TypeORM`. I guess not really since if we switch ORMs we wouldn't want to have 2 orm packages installed.

```suggestion
  orm?: MikroORM
```

Reply:
That object has to be compatible with Mikro's `RequestContext`, so it can only be a `MikroORM`

It's an unfortunate bit of coupling where our http layer has to know about a specific persistence layer for the sake of performance